packages/console/src/log/LogView.tsx

```diff
@@ -211,6 +211,7 @@ class LogView extends PureComponent<LogViewProps, LogViewState> {
       fixedOverflowWidgets: true,
       folding: false,
       fontFamily: 'Fira Mono',
+      lineNumbers: 'off',
```

Comment:
Can probably delete this comment now:

```
      // I commented this out since '' is not a valid parameter for line Numbers
      // lineNumbers: '',
 ```
 
 Looks like this was switched off during TS migration, rather than looked up.